frontend/src/apis/index.js

```diff
@@ -79,5 +79,6 @@ export const reserveTicket = ({ courseId, eventId }) =>
 
 export const fetchReservedCourses = () => myFetch(`/reservations`);
 
-export const requestAddBalance = (amount) =>
-    myFetch(`/payments/add-balance?amount=${amount}`);
+export const requestAddBalance = async (amount) => {
```

Comment:
aye this will not the the function return a promise. If you wait for the promise here then also attach a try catch :D, but as you see there are real error handling in the methods of the stores, so you would have to propagate an error

This could have easily been avoided using promise.
Plus you can actually set some state while you wait on the method of the store, instead of wait here

Reply:
removed it.